projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/breakpoint.service.migration.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  BREAKPOINT_SERVICE,
+  GET_BREAKPOINT,
+  GET_CLOSEST,
+  GET_WINDOW,
+  SPARTACUS_STOREFRONTLIB,
+  TODO_SPARTACUS,
+  WINDOW_REF,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+// projects/storefrontlib/src/layout/breakpoint/breakpoint.service.ts
+export const BREAKPOINT_SERVICE_MIGRATION: MethodPropertyDeprecation[] = [
```

Comment:
This `BREAKPOINT_SERVICE_MIGRATION` should be registered in `methods-and-properties-deprecations.ts`